### ISSUE TITLE
Group filtering

### ISSUE THREAD

**Issue by maro2511** [TextAnalysisTool/Releases#121]
Hi!
I saw this feature was requested many times in the past, adding my request also because I think it's the ultimate missing feature:
"Add option to group multiple filters so that you can enable/disable the whole group of with one checkbox click."
Thanks!

**Comment by vincec-msft** [comment]
Not quite the same, but as a simple workaround: you can enable/disable multiple filters at once by selecting them and clicking any of their checkboxes. It's not 1 click, but you can do any number of filters with 3 clicks: select the first, shift+click the last, click the checkbox.

**Comment by maro2511** [comment]
I tried that. If you select more than 3 filters and then click on the checkbox to enable/disable them together, it will only affect the first filter and the one you clicked on (ignoring the filters in between).
Having groups will also "free" letters for quick lookup. I use 60+ filters so I constantly scrolling to enable/disable some of them.

**Comment by vincec-msft** [comment]
You're right about the multi-selection behavior. It's improved in the next version. :)
Is the grouping request covered by #116?

**Comment by maro2511** [comment]
I think so, yes

**Comment by maro2511** [comment]
@vincec-msft  Any plan to release a new version?

**Comment by vincec-msft** [comment]
Hi @maro2511. Yes! We are planning a release. It won't be soon, but your question has gotten the ball rolling. Thanks!

**Comment by armar271** [comment]
@vincec-msft  Checking if there is any update about the new release (with group filtering) :)

**Comment by vincec-msft** [comment]
@armar271 and @maro2511, Version 2025-06-17 is available at https://textanalysistool.github.io/.

The multi-select behavior is improved. Still no formal grouping of filters. That's tracked in #116 so I'm closing this issue.
